modules/rostests/rosautotest/main.cpp

```diff
@@ -9,6 +9,8 @@
 #include <cstdio>
 #include <ndk/setypes.h>
 #include <ndk/exfuncs.h>
+#include <strsafe.h>
+WCHAR TestName[MAX_PATH];
 
```

Comment:
```suggestion

WCHAR TestName[MAX_PATH];
```

Reply:
Now there is a blank line above the "WCHAR..." line. Thanks.